(Part - 1) Add user's web app settings to accounts.susi

PrP-11 opened the pull request:
Fixes #266 

Changes: 
The work on issue #266 is divided into two parts. Part - 1 is covered in this PR.

**Part - 1**
- Display user's accounts and web client settings.

**Part - 2**
- Enable edit and save changes for settings.
- Implement similar UI as on chat.susi settings.

Surge Deployment Link: http://nondescript-shop.surge.sh/

Screenshots for the change:
![settings](https://user-images.githubusercontent.com/30981465/41202459-dde520c0-6ce6-11e8-8d73-c4197aefaccd.png)

hkedia321 commented:
@PrP-11 For the points 1-4, it works fine when the `prefLanguage` parameter comes from the API. However for some users, this parameter might be missing, so please handle such cases.

mariobehling commented:
Please use the same style as in chat.susi.ai settings, e.g. left side bar is not gray here.

PrP-11 commented:
@hkedia321 I have fixed the bug. Thanks for pointing that out. I have also removed the warnings and errors of the console. Regarding the nav bar for mobile view, I am working on it and can be fixed in part - 2. Also the dropdown menu is positioned at bottom already for me. Please check that again and if it still persists please let me know about your browser. Thanks.

@DNS-404 Regading the country code, it is given twice on chat.susi also. I myself was confused to whether show it or not, but now I have removed it now. Thanks.

anshumanv commented on src/components/Settings/Settings.react.js:
```diff
@@ -1,36 +1,455 @@
+			dataType: 'jsonp',
+			statusCode: {
+				404: function(xhr) {
+					if(window.console) {console.log(xhr.responseText);}
```
Can you please explain about the use case of this log?

PrP-11 replied:
Whenever the user settings are not applicable for a user role or the settings are not found on the accounting model then this error occurs. This is implemented the same way as on chat.susi settings. Thanks.

anshumanv commented on src/components/Settings/Settings.react.js:
```diff
@@ -1,36 +1,455 @@
+			timeout: 3000,
+			async: false,
+			success: function (response) {
+				console.log(response.settings);
```
Can be removed

anshumanv commented on src/components/Settings/Settings.react.js:
```diff
@@ -1,36 +1,455 @@
+    if(!this.state.dataFetched && cookies.get('loggedIn')){
+      this.apiCall();
+    }
+    if(this.state.dataFetched && cookies.get('loggedIn')){
```
This block can be removed.

PrP-11 commented:
@anshumanv I will mimic the settings on chat.susi in the part 2 itself. Currently this can be merged since currently the settings page is empty and I will create PR for part 2 as soon as its complete. Thanks.

@hkedia321 @DNS-404 @anshumanv @akshatnitd @Akshat-Jain @AnupKumarPanwar Please review.
